common/build.gradle.kts

```diff
@@ -40,27 +40,35 @@ android {
     buildFeatures {
         viewBinding = true
     }
-    namespace = "com.redmadrobot.debug_panel_common"
+    namespace = "com.redmadrobot.debug.panel.common"
 }
 
 dependencies {
-    api(kotlin("stdlib"))
+    api(androidx.compose.animation)
```

Comment:
It looks like 8dc99046d156962b421e556a72627ec821799a79 should be excluded from this branch